service/GroupService.js

```diff
@@ -52,10 +52,11 @@ exports.deleteGroup = function(groupID) {
 exports.enrollStudent = function(body) {
   return new Promise(function(resolve, reject) {
     var examples = {};
-    examples['application/json'] = {
-  "studentID" : 0,
-  "groupID" : 6
-};
+    // examples['application/json'] = {
+    //   "studentID" : 0,
+    //   "groupID" : 6
+    // };
+    examples['application/json'] = body;
```

Comment:
This is to have some actuall logic on the server and not always return the example no matter how messed up the query (so the tests have some meaning).